Formula/jack.rb

```diff
@@ -24,7 +25,7 @@ class Jack < Formula
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "python@3.10" => :build
-  depends_on "berkeley-db"
+  depends_on "berkeley-db@5" # berkeley-db > 5 is AGPL, so keep it at version 5 to avoid relicense
```

Comment:
This will need an audit in homebrew, comments can't be relied on.

Reply:
So shall I remove this?